lib/helpers/get-form-view-model.js

```diff
@@ -12,13 +11,20 @@ var cacheTimestamp = 0;
  *
  * @method
  *
+ * @param {string} formName - The name of the form (e.g. 'login' or 'register').
  * @param {number} ttl - The time-to-live value for the cached object.
  */
-function getCachedModel(ttl) {
-  var age = (new Date().getTime() - cacheTimestamp) / 1000; // In seconds.
+function getCachedModel(formName, ttl) {
+  var cacheItem = cache[formName];
 
-  if (cachedViewModel && age <= ttl) {
-    return cachedViewModel;
+  if (!cacheItem) {
+    return null;
```

Comment:
Minor nitpick. I'd rather return false here instead of null. Since `false` would indicate that there isn't anything.